examples/express/app.js

```diff
@@ -0,0 +1,35 @@
+const bugsnag = require('@bugsnag/js')
+const express = require('express')
+const { readFileSync } = require('fs')
+
+const bugsnagClient = bugsnag(process.env.BUGSNAG_API_KEY)
+bugsnagClient.use(require('@bugsnag/plugin-express'))
+
+const app = express()
+const { requestHandler, errorHandler } = bugsnagClient.getPlugin('express')
+
+app.use(requestHandler)
+
+app.use('/static', express.static(`${__dirname}/static`))
+
+const index = readFileSync('./views/index.html', 'utf8')
+app.get('/', (req, res) => {
+  res.send(index)
+})
+
+app.post('/crash', (req, res) => {
+  setTimeout(() => { throw new Error('Uh oh') })
```

Comment:
Is there a way we can either recover or notify the user they need to restart the server on visiting this route?

Reply:
Or have some indication that this route should be visited last?  Especially for the dockerised version

Reply:
Good point! I reorderd the buttons and put a note in the readme.